### ISSUE TITLE
issue with accelerate

### ISSUE THREAD

**Issue by Abbyhub3** [bmaltais/kohya_ss#3072]
i tried to use pip install accelerate, and did all updates and installs, But everytime i press train i get this error 

21:02:13-983148 INFO     Start training Dreambooth...
21:02:13-984147 INFO     Validating lr scheduler arguments...
21:02:13-986146 INFO     Validating optimizer arguments...
21:02:13-988145 INFO     Validating R:/Stable_diffusion/train/these/log existence and writability... SUCCESS
21:02:13-989143 INFO     Validating R:/Stable_diffusion/train/these/model existence and writability... SUCCESS
21:02:13-991142 INFO     Validating runwayml/stable-diffusion-v1-5 existence... SKIPPING: huggingface.co model
21:02:13-992141 INFO     Validating R:/Stable_diffusion/train/these/image existence... SUCCESS
21:02:13-994139 INFO     Folder 100_Hela: 100 repeats found
21:02:13-995139 INFO     Folder 100_Hela: 36 images found
21:02:13-998137 INFO     Folder 100_Hela: 36 * 100 = 3600 steps
21:02:13-999136 INFO     Regulatization factor: 1
21:02:14-000135 INFO     Total steps: 3600
21:02:14-001135 INFO     Train batch size: 1
21:02:14-002134 INFO     Gradient accumulation steps: 1
21:02:14-004133 INFO     Epoch: 1
21:02:14-005131 INFO     Max train steps: 1600
21:02:14-006131 INFO     lr_warmup_steps = 160
21:02:14-048098 ERROR    accelerate not found"

I dont know what else i can do..

also i didnt install cudn because i keep getting this error
"ERROR    Installation Failed: "R:\Stable_diffusion\train2\kohya_ss\venv\Lib\site-packages\nvidia\cudnn\bin"
                         could not be found."

can someone pleassse help me out here

**Comment by Sylsatra** [comment]
"ERROR Installation Failed: "R:\Stable_diffusion\train2\kohya_ss\venv\Lib\site-packages\nvidia\cudnn\bin"
You are missing this file so go to ChatGPT or DeepSeek R1, and paste that error with your cuda version and it will teach you how to resolve this error!

**Comment by b-fission** [comment]
Did the path to your kohya_ss folder change recently? Anything like moving/renaming the folder, or changing drive letters could break the python wrapper for accelerate and dependencies.

> i tried to use pip install accelerate, and did all updates and installs

Did you run pip from the venv (and not the system's python)? Try uninstalling accelerate with this command in the kohya_ss folder: `venv\scripts\pip.exe uninstall -y accelerate`
Then start the gui again to auto-install the required version of accelerate.
